jinja2_standalone_compiler/__init__.py

```diff
@@ -4,14 +4,23 @@
 import os
 import re
 import sys
-
 import click
+from colorama import init, Fore, Style
```

Comment:
> Does it really matter when receiving the lib trought pip?
Yes it does. Colorama is somewhat not needed for this to work. (worked until now without it).

> It should install colorama, no?
It would install the way you did, but shouldn't. For instance, take a look at [1].

> How can make an optional requirement?
A good approach can be found on [2].

> I think I need colorama in requiremets.txt to build the egg, right?
If it's wrapped in a `try..except` clause it's not needed on the egg.

[1] https://github.com/worldveil/deuces
> If you have termacolor installed, they will be colored as well.

[2] https://github.com/worldveil/deuces/blob/master/deuces/card.py#L172-L179